src/inheritance.js

```diff
@@ -0,0 +1,30 @@
+
+function Rectangle (width, height) {
+  this.width = width
+  this.height = height
+}
+
+Rectangle.prototype.area = function () {
+  return this.width * this.height
+}
+
+Rectangle.prototype.description = function () {
+  return 'Rectangle of width ' + this.width + ', height ' + this.height + ' and area ' + this.area() + '.'
+}
+
+function Square (side) {
+  this.side = side
+  this.width = side
+  this.height = side
```

Comment:
Puedes mejorar el constructor de Square con una llamada al constructor de la clase padre.

Relevant: http://wildlyinaccurate.com/understanding-javascript-inheritance-and-the-prototype-chain/